src/linux/component.rs

```diff
@@ -9,152 +14,392 @@ use std::path::{Path, PathBuf};
 
 #[doc = include_str!("../../md_doc/component.md")]
 pub struct Component {
-    temperature: f32,
-    max: f32,
-    critical: Option<f32>,
+    /// Optional Associated device of a Component
+    device_model: Option<String>,
+    /// The chip name.
+    /// 	This should be a short, lowercase string, not containing
```

Comment:
This part of the doc comment will be rendered as a rust code block, which is not what you want I assume. Please remove the indent and put an empty line before it.